Sateltiles/admin_user.py

```diff
@@ -0,0 +1,147 @@
+        self.SetIcon(self.icon)
+
+        font = wx.SystemSettings_GetFont(wx.SYS_SYSTEM_FONT)
+        font.SetPointSize(25)
+        existing_label = wx.StaticText(panel, pos=(20,20), label='Existing TMS')
+        existing_label.SetFont(font)
+
+        generate_label = wx.StaticText(panel, pos=(20,200+130), label='Generate TMS')
+        generate_label.SetFont(font)
+
+        source_filePath_label = wx.StaticText(panel, pos=(10,230+130), label='Source file (.tif) : ') 
+        self.source_filePath_entry = wx.TextCtrl(panel, pos=(100,230+130), size=(165,20)) # sorce file path entries
+
+        save_filePath_label = wx.StaticText(panel, pos=(20,260+130), label='Save location : ')
+        self.save_filePath_entry = wx.TextCtrl(panel, pos=(100,260+130), size=(165,20)) # destination file path entries
+
+        getLink_button = wx.Button(panel, label='GET TMS LINK', pos=(470,190+130), size=(100,40)) 
+        getLink_button.Bind(wx.EVT_BUTTON, self.getTMS_Link)
+
+        back_button = wx.Button(panel, label='BACK', pos=(10,310+130), size=(100,40)) #back home buttons
+        back_button.Bind(wx.EVT_BUTTON, self.backHome)
+
+        source_button = wx.Button(panel, label='Browse', pos=(280,227+130), size=(90,25)) 
+        source_button.Bind(wx.EVT_BUTTON, self.browseSourceFile)
+
+        save_button = wx.Button(panel, label='Browse', pos=(280,257+130), size=(90,25)) 
+        save_button.Bind(wx.EVT_BUTTON, self.browseSaveFile)
+
+        generate_button = wx.Button(panel, label='GENERATE', pos=(175,285+130), size=(90,30)) 
+        generate_button.Bind(wx.EVT_BUTTON, self.generateTMS)
+
+        id=wx.NewId()
+        self.list = wx.ListCtrl(panel,pos=(20,0+50), size=(550,260), style=wx.LC_REPORT 
+                                 | wx.BORDER_SUNKEN
+                                 #| wx.BORDER_NONE         <<< This is up to your style uncomment # to enable
+                                 #| wx.LC_EDIT_LABELS
+                                 #| wx.LC_SORT_ASCENDING
+                                 #| wx.LC_NO_HEADER
+                                 #| wx.LC_VRULES
+                                 #| wx.LC_HRULES
+                                 #| wx.LC_SINGLE_SEL
+                                  )
+        
+
+        self.list.InsertColumn(0,"Min lat")
+        self.list.InsertColumn(1,"Max lat")
+        self.list.InsertColumn(2,"Min long")
+        self.list.InsertColumn(3,"Max long")
+        self.list.InsertColumn(4,"EPSG")
+        self.list.InsertColumn(5,"Create by")
+        self.list.InsertColumn(6,"Create on")
+        self.list.InsertColumn(7,"Hide")
+
+
+        for i in range(100):
+            pos = self.list.InsertStringItem(i,"80") # 0 will insert at the start of the list
+            self.list.SetStringItem(pos,1,"26.45") # add values in the other columns on the same row
+            self.list.SetStringItem(pos,2,"88.183333")
+            self.list.SetStringItem(pos,3,"30.45")

```

Comment:
inconsistent styling..you are using " which is preferred
put a single space after comma(,) like self.list.SetStringItem(pos, 3, "30.45") not self.list.SetStringItem(pos,3,"30.45")